libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/PParams.hs

```diff
@@ -3,17 +3,23 @@
 
 module Test.Cardano.Ledger.Constrained.Conway.PParams where
 
-import Cardano.Ledger.BaseTypes
-import Cardano.Ledger.Conway.PParams
-
-import Constrained
-
 import Cardano.Ledger.Conway (ConwayEra)
 import Cardano.Ledger.Conway.Core
 import Cardano.Ledger.Crypto (StandardCrypto)
+import Constrained
+
+pparamsSpec ::
+  Specification fn (PParams (ConwayEra StandardCrypto))
+pparamsSpec = TrueSpec
+
+{-
 import Test.Cardano.Ledger.Constrained.Conway.Instances (IsConwayUniv)
+import Cardano.Ledger.BaseTypes
+import Cardano.Ledger.Conway.PParams
 
-pparamsSpec :: IsConwayUniv fn => Specification fn (PParams (ConwayEra StandardCrypto))
+pparamsSpec ::
```

Comment:
Not sure about keeping around this function commented out like this. I would prefer to either remove it, or rename it to start with underscore and maybe explain why it's not used in a comment - otherwise it might diverge  and also maybe get us confused after a while, when we would have forgotten what was going on here. 
But maybe there is some good reason for it, it's not a big deal either way.

Reply:
Thank you for this idea. I will remove the comments and just use an underscore. :raised_hands: This way it continues to be type-checked.